easybuild/easyconfigs/g/grpcio/grpcio-1.57.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'grpcio'
+version = '1.57.0'
+
+homepage = 'https://grpc.io/'
+description = """gRPC is a modern, open source, high-performance remote procedure call (RPC)
+framework that can run anywhere. gRPC enables client and server applications to
+communicate transparently, and simplifies the building of connected systems."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+use_pip = True
+sanity_pip_check = True
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+    ('RE2', '2023-08-01'),
```

Comment:
Just to check: are you sure RE2 is only a build dependency for `grpcio`? I couldn't quickly figure this out myself...

Reply:
Thanks, wasn't careful enough when commiting, but, to my understanding, RE2 is only build dep -- there is only `libre2.a` in `RE2/2023-08-01-GCCcore-12.3.0/lib` and `ldd cygrpc.cpython-311-x86_64-linux-gnu.so` doesn't show anything related to re2, so I reckon it's all statically linked.

On the other hand I missed that Abseil is actually a dependency but that should be loaded already via `python-protobuf->profobuf`, so I think I shall remove that from build deps?

Reply:
If it's also a dep of `grpcio`, please also include it here (even if it would already be loaded as a dep of a dep). We conventionally do that since it is the most explicit, and has the added benefit that if ever `protobuf` stops depending on `Abseil` we don't suddenly need to adapt the `grpcio` EasyConfigs.